### PR TITLE
Save audio file to the CSAI Google Drive

### PR DESCRIPTION
**Pull request by snekiam**
All the IDs and keys are not being uploaded to github for security reasons.  This addresses #8.

### CODE REVIEW

File: flask_api.py [hunk 1]
```diff
@@ -90,11 +93,26 @@ def resample_audio():
...
-def save_audiofile():
-    """Actually save the file into Google Drive or whereever we may do it."""
-    pass
-
+def save_audiofile(filename, content):
+    """Actually save the file into Google Drive."""
+    # Initialize our google drive authentication object using saved credentials,
+    # or through the command line
+    gauth = GoogleAuth()
+    gauth.CommandLineAuth()
```
Comment (Waidhoferj) [inline flask_api.py:101]: Does this require command line input? Could we pipe in a credentials.txt file to automate the process?

**Review by Waidhoferj — APPROVED** [review]

**Comment by Waidhoferj** [comment]
Though I can't verify through direct testing, the code looks great! We should probably loop in @mfekadu to push the code to our live server!

File: flask_api.py (hunk 1 above)
Comment (snekiam) [reply flask_api.py:101]: It does if you don't have a credentials.json and a settings.yaml file, which contain the client secret as well as an account token so I didn't upload them here.

Comment (Waidhoferj) [reply flask_api.py:101]: Okay sounds good! Later we can work those into Github Actions under a ``secret``

Comment (snekiam) [reply flask_api.py:101]: Basically every file will be uploaded/owned by the csaicalpoly gmail account or one of our choosing.

Comment (Waidhoferj) [reply flask_api.py:101]: That's ideal, all of the user info will be included in the filename.

**Comment by mfekadu** [comment]
Super cool! Will look into this soon! Thanks @snekiam

**Pull request merged**